config/config_test.go

```diff
@@ -37,20 +37,20 @@ var configPath = flag.String("config", "config.yaml", "Path to generated config"
 var cfg org.FullConfig
 
 func TestMain(m *testing.M) {
-	flag.Parse()
-	if *configPath == "" {
-		fmt.Println("--config must be set")
+	configPath, exists := os.LookupEnv("CONFIG_PATH")
+	if !exists {
+		fmt.Println("Environment variable \"CONFIG_PATH\" must be set")
 		os.Exit(1)
 	}
 
-	raw, err := ioutil.ReadFile(*configPath)
+	raw, err := ioutil.ReadFile(configPath)
 	if err != nil {
-		fmt.Printf("cannot read generated config.yaml from %s: %v\n", *configPath, err)
+		fmt.Printf("cannot read generated config.yaml from %s: %v\n", configPath, err)
```

Comment:
@mohitsharma-in, nit:

remove the following unused bits of code in `config/config_test.go`

- unused package import - "flag"

- this is no longer required:

```go
var configPath = flag.String("config", "config.yaml", "Path to generated config")
```